### PR TITLE
feat: 💥 change tag api to batch

### PR DESCRIPTION
**Pull request by binsee**
#### Checklist

- [ ] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
- [ ] Tests for the changes have been added
- [ ] CI has been passed. (GitHub actions all turns green)
- [ ] CLA has been signed

### CODE REVIEW

File: proto/wechaty/puppet/tag.proto [hunk 1]
```diff
@@ -55,30 +55,38 @@ message TagGroupDeleteRequest {
...
+message TagTagNewInfo {
+  string tag_id   = 1;
+  string tag_new_name = 2;
+}
+
 message TagTagAddRequest {
-  string tag_name     = 1;
-  string tag_group_id = 2;
+  string   tag_group_id         = 2;
+  repeated string tag_name_list = 3;
```
Comment (su-chang) [inline proto/wechaty/puppet/tag.proto:70]: 这个tag_name_list应该会被自动再加一次list，可以考虑去掉，下面两个同理。

Comment (binsee) [reply proto/wechaty/puppet/tag.proto:70]: 啊？编译框架自动搞这个？我试试

Comment (binsee) [reply proto/wechaty/puppet/tag.proto]: 确实如此，现有定义中还有一些名称是有点小问题的
`xxIds`  => `xxIdsList`
image
image

File: proto/wechaty/puppet/tag.proto [hunk 2]
```diff
@@ -55,30 +55,38 @@ message TagGroupDeleteRequest {
 message TagGroupDeleteResponse {
 }
 
+message TagTagInfo {
+  string tag_id   = 1;
+  string tag_name = 2;
+}
+
+message TagTagNewInfo {
```
Comment (su-chang) [inline proto/wechaty/puppet/tag.proto:63]: 这个也复用上面的 TagTagInfo 吧，多一个 TagTagNewInfo 有点奇怪。

File: proto/wechaty/puppet/tag.proto [hunk 3]
```diff
@@ -55,30 +55,38 @@ message TagGroupDeleteRequest {
...
+  string tag_id   = 1;
+  string tag_new_name = 2;
+}
+
 message TagTagAddRequest {
-  string tag_name     = 1;
-  string tag_group_id = 2;
+  string   tag_group_id         = 2;
```
Comment (su-chang) [inline proto/wechaty/puppet/tag.proto:69]: 为啥把1跳过了，是改叉劈了么，虽然不影响。

**Review by su-chang — APPROVED** [review]

File: proto/wechaty/puppet/tag.proto (hunk 3 above)
Comment (binsee) [reply proto/wechaty/puppet/tag.proto]: 因为改成了数组，虽然 proto 解析不会出问题，但生成的 ts 会与以前不兼容。

**Pull request merged**